src/utils/httpUtils.ts

```diff
@@ -0,0 +1,13 @@
+export function getURL(baseURL: string, path: string, orgId?: string): string {
+  if (routeToGateway(baseURL)) {
+    if (!orgId) {
+      throw new Error('Org is required for this operation');
+    }
+    return `${baseURL}/hidden/orgs/${orgId}/code${path}`;
+  }
+  return `${baseURL}${path}`;
+}
+
+function routeToGateway(baseURL: string): boolean {
+  return baseURL.includes('snykgov.io');
```

Comment:
So, when all deeproxy changes will be rolled out to all envs, this check will become redundant?

Reply:
Yes.
We will remove this check and use the new way for all the routes.
Old versions will still call with the old way but we have a plan on how we support the 2 situations